### PR TITLE
Create pro1_qwerty_svk_1.kcm

### PR DESCRIPTION
**Pull request by sorgo**
Pure theoretical attempt.
Need to validate the key codes mappings of actual keys on the pro1 once possible.
The official layout is QWERTZ, but many people (including myself) that grew up on the US keyboard prefer the QWERTY one, which I guess will be the case of pro1 owners. Once validated, we can add the QWERTZ.

### CODE REVIEW

**Comment by anssih** [comment]
Oops, sorry, it seems I was not receiving notifications for pull requests.

I will check this next week :)

**Comment by sorgo** [comment]
No worries, guess what, my device didn't arrive yet anyway.
I fear this keymap is useless and the device has an ootb mapping. Please, if you can check if it's qwerty or qwertz or both for Slovak input map. I would definitely want to have a qwerty layout. Thanks

File: finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+map key 46 V
+map key 47 B
+map key 48 N
+map key 49 M
+map key 50 COMMA
+map key 51 PERIOD
+map key 52 SLASH
+
+# orig BACK
+#map key 158 ESCAPE
```
Comment (anssih) [inline finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm]: I think we prefer escape key to actually be ESCAPE, so I'd keep this in.

But reportedly it was going to be changed to ESCAPE in the stock layout too, so maybe this bit is unnecessary now. We'll see when we get the actual devices.

File: finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+    label:                              'P'
+    base:                               'p'
+    shift, capslock:                    'P'
+}
+
+key LEFT_BRACKET { 
+    label:                              '\u00fa' # ú
+    base:                               '\u00fa' # ú
+    shift:                              '/'
+    capslock:		                    '\u00c5' # Ú
```
Comment (anssih) [inline finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm]: Different indentation than others.

**Review by anssih — CHANGES_REQUESTED** [review]
Looks pretty good. And better comments than my .kcms usually have :)

It is missing adding the kcm to finqwerty/src/main/res/xml/finqwerty_layouts.xml to get it actually listed, and the commit message has too long lines, so those need to be fixed before merging (if we are going to do that).

**Comment by anssih** [comment]
> No worries, guess what, my device didn't arrive yet anyway.
> I fear this keymap is useless and the device has an ootb mapping. Please, if you can check if it's qwerty or qwertz or both for Slovak input map. I would definitely want to have a qwerty layout. Thanks

I don't actually have the device yet either, so I can't check the layouts it has. The other Pro1 layouts were tested by Eske Rahn.

**Comment by anssih** [comment]
The PR was autoclosed as the obsolete pro1 branch was removed, reopened and switched to target master now.

File: finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm (hunk 1 above)
Comment (sorgo) [reply finqwerty/src/main/res/raw/pro1_qwerty_svk_1.kcm:54]: I think this one is resolved by now :)

**Review by anssih — APPROVED** [review]

**Pull request merged**